ports/spdlog/vcpkg.json

```
@@ -0,0 +1,25 @@
+{
+  "name": "spdlog",
+  "version": "1.8.3",
```

```suggestion
  "version-semver": "1.8.3",
```
It might be better to use `version-semver` here. Please see https://github.com/microsoft/vcpkg/blob/master/docs/specifications/versioning.md#22-package-versions